py/configu/stores/json_file_store.py

```diff
@@ -17,60 +10,44 @@ class JsonFileConfigStore(ConfigStore):
     _path: str
 
     def __init__(self, path: str) -> None:
-        super().__init__(type="in-memory")
+        super().__init__(type="json-file")
         self._path = path
 
-    def read(self):
+    def _read(self) -> List[Config]:
         """
         Reads JSON file contents
-        :return: The file contents, represented as a list of key/set/value
+        :return: The file contents, represented as a list of Config
         dicts
         """
         with open(self._path, mode="r", encoding="utf-8") as json_file:
             data = json.load(json_file)
-        return config_store_contents_from_dict(data)
+        return [Config(**args) for args in data]
```

Comment:
Yes. doing my best to get rid of convertors from generated.py

Reply:
@RonConfigu is Config is part of the generated code?
`quicktype` is used to enforce the base interfaces of all the SDKs so I wonder, do you guys think it is irrelevant for python or do you use it in some other parts without the convertors?

Reply:
yup, both versions are provided by the `quicktype` generated code. This is simpler however since it doesn't rely on an auxillary alias of `ConfigStoreContentsElement`, nor does it rely on a utility `config_store_contents_from_dict` function. In theory this takes us one step closer to entirely removing some of the unnecessary auxillary types from `quicktype` entirely.